### PR TITLE
Removes z-index from the avatar

### PR DESCRIPTION
**Pull request by mike-lvov**
Fixes the reaction layering.

This regression was  introduced by `nametag` PR https://github.com/sparkletown/sparkle/pull/1049

### CODE REVIEW

**Comment by goran-peoski-work** [comment]
LGTM

File: src/components/atoms/UserAvatar/UserAvatar.scss [hunk 1]
```diff
@@ -2,18 +2,13 @@
 
 .UserAvatar {
   position: relative;
-  z-index: z(global-profile-icon);
   width: 25px;
   height: 25px;
 
   &:hover {
-    z-index: z(global-profile-icon--hover);
```
Comment (denisdimitrov) [inline src/components/atoms/UserAvatar/UserAvatar.scss:10]: If these are not used anywhere else, please remove them.

Comment (mike-lvov) [reply src/components/atoms/UserAvatar/UserAvatar.scss:10]: One is used in Convers. space
image

Removed the other in https://github.com/sparkletown/sparkle/pull/1808/commits/b028120d2984ad6df18f4a3167cb10712e545ab3

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**